lib/gui/app/components/settings/settings.tsx

```diff
@@ -37,6 +37,10 @@ async function getSettingsList(): Promise<Setting[]> {
 			name: 'errorReporting',
 			label: i18next.t('settings.errorReporting'),
 		},
+		{
+			name: 'autoBlockmapping',
+			label: 'Trim ext{2,3,4} partitions before writing (raw images only)',
```

Comment:
Looking at the lines above and below this change, should the label-text here have been using `i18next.t()` ? :man_shrugging:

Reply:
good call @lurch